server/src/repositories/asset.repository.ts

```diff
@@ -532,6 +532,20 @@ export class AssetRepository {
       .execute();
   }
 
+  @GenerateSql({ params: [DummyValue.UUID, 5] })
+  getLargeAssets(ownerId: string, take: number) {
+    return this.db
+      .selectFrom('assets')
+      .selectAll('assets')
+      .$call(withExif)
+      .where('ownerId', '=', ownerId)
+      .where('isVisible', '=', true)
```

Comment:
#17939 Will conflict here , I think

Reply:
Actually, I think this filter can be removed completely? Archived assets should also be shown when seraching for large files

Reply:
removed with [d911591](https://github.com/immich-app/immich/pull/18040/commits/d911591eb5a78478e28e347cc5765ba3af4daaaa)